TASVideos/wwwroot/css/partials/_customizations.scss

```diff
@@ -282,3 +282,9 @@ article.wiki, .postbody {
 .fa {
 	transform: translate(0);
 }
+
+.card-header {
```

Comment:
this affects every card on the site, are you sure this makes sense?

Reply:
Yes!
The card-header tag itself already has `margin-bottom: 0;` and it was meant to be used as a tag for h1-h6, not as a class for a div with h1-h6 inside. They overwrite the `margin-bottom: 0;` and are not centered anymore. But since using it in a div makes stuff like buttons and badges possible, this is the workaround.